### PR TITLE
Update Python developer guides to document how to get the correct Consul/Vault prefix.

### PR DESCRIPTION
**Pull request by hagraham**

### CODE REVIEW

**Review by mtsizis — APPROVED** [review]
+1

File: 08-python-hello-world-service-example/04-adding-consul-configuration.md [hunk 1]
```diff
@@ -93,20 +93,39 @@ import pkgutil
...
         config["consul"]["port"] = environ.get("SPRING_CLOUD_CONSUL_PORT", config["consul"]["port"])
         self.consul = ConsulConfig(**config["consul"])
+
+    # Find the correct Consul/Vault prefix.
+    def find_consul_vault_prefix(self, consul_helper):
+        try:
+            test_value = consul_helper.get_string("thirdpartycomponents/defaultapplication/swagger.security.sso.baseUrl", None)
+        except ACLPermissionDenied:
+            test_value = None
+        self._config_prefix = "thirdpartycomponents" if test_value else "thirdpartyservices"
```
Comment (v0lkan) [inline 08-python-hello-world-service-example/04-adding-consul-configuration.md:123]: 👍

**Review by v0lkan — APPROVED** [review]
Looking good 🌮 .

File: 08-python-hello-world-service-example/04-adding-consul-configuration.md (hunk 1 above)
Comment (mtsizis) [reply 08-python-hello-world-service-example/04-adding-consul-configuration.md:123]: LGTM

**Pull request merged**